src/test/TestField.js

```diff
@@ -28,23 +28,23 @@ type Props = {|
   validation: Validation<string>,
 |};
 
-export default class TestField extends React.Component<Props> {
-  static defaultProps = {
-    validation: alwaysValid,
-  };
-
-  render() {
-    return (
-      <Field link={this.props.link} validation={this.props.validation}>
-        {(value, errors, onChange, onBlur) => (
-          <TestInput
-            value={value}
-            errors={errors}
-            onChange={onChange}
-            onBlur={onBlur}
-          />
-        )}
-      </Field>
-    );
-  }
+const TestField = (props: Props) => {
```

Comment:
nit: We prefer the JS-native default parameters to `defaultProps`.

```suggestion
const TestField = (props: Props) => {
function TestField({link, validation = alwaysValid}: Props) {
```